src/test/resources/cwl/oneline_optional_types.cwl

```diff
@@ -5,7 +5,12 @@ inputs:
   ncrna_tab_file: {type: File?}
   reverse_reads: File?
   qualified_phred_quality: { type: int? }
-  
+  ssu_tax: [string, File]
+  rfam_models:
+    type:
+      - type: array
+        items: [string, File]
+
```

Comment:
Nice. This test appears to contain both options for declaring the values, with and without the `?` char :+1: